app/modules/conversations/conversations_router.py

```
@@ -105,16 +113,22 @@ async def post_message(
     async def regenerate_last_message(
         conversation_id: str,
         request: RegenerateRequest,
+        stream: bool = Query(True, description="Whether to stream the response"),
         db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)